installer/client/src/pages/CurrentVersionScreen.vue

```diff
@@ -0,0 +1,139 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+<template>
+  <oxd-form class="orangehrm-current-version">
+    <oxd-text tag="h5" class="orangehrm-current-version-title">
+      Current Version Details
+    </oxd-text>
+
+    <oxd-text class="orangehrm-current-version-content">
+      Select your current OrangeHRM version here.You can't find the version at
+      the bottom of OrangeHRM login page.OrangeHRM upgrader only supports
+      versions listed in the dropdown selecting a different version would lead
+      to an upgrader failure and a database corruption.
+    </oxd-text>
+
+    <oxd-form-row class="orangehrm-current-version-row">
+      <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+        <oxd-grid-item>
+          <oxd-input
+            v-model="config.version"
+            type="select"
+            label="Current OrangeHRM Version"
+            :options="getItems"
```

Comment:
Computed property is not a function, its a property so **get**Items name is bad and computed property should be used for reactive purposes and transformations. here the versions are loaded only once from BE why do you need a computed property for it? 

```
getItems() {
      if (this.items.length == 0) {
        return [];
      }
      return JSON.parse(JSON.stringify(this.items)); // why ?
    },
```
The flow should be.

> 1.  Declare a state variable for version items
> 2.  Before component load, send a http request to get verison items from BE
> 3.  Assign fetched data to state variable
> 4.  Pass the state variable to dropdown component as props

Check how dropdown options are loaded in other places of app. 
https://github.com/orangehrm/orangehrm/blob/5.0/symfony/client/src/orangehrmPimPlugin/components/JobtitleDropdown.vue